mlir-tensorrt/executor/lib/Runtime/API/API.cpp

```diff
@@ -725,9 +731,16 @@ StatusOr<std::unique_ptr<MemRefValue>> MemRefValue::create(
   if (!::getFootprintInBytes(shape, strides, bitsPerElement).isOk())
     return getInvalidArgStatus(
         "only memrefs with non-negative strides are allowed");
-  if (!ptr)
+
+  auto is_empty_tensor = [](const llvm::ArrayRef<int64_t> &shape) -> bool {
```

Comment:
Pass `llvm::ArrayRef` by value, not by reference. `const` also doesn't add anything of value when passing ArrayRef, so the convention that's always used is to pass a plain non-const `ArrayRef<T>` by value.